.github/actions/mlnx/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM centos:7.6.1810
 
```

Comment:
Please add `RUN yum install -y epel-release` here.

Reply:
That got it.  Thanks!